samples/bookinfo/kube/rbac/rbac-config-ON.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: RbacConfig
+metadata:
+  name: default
```

Comment:
Do we want to put it in istio-system namespace?

Reply:
I think it makes no difference, the `RbacConfig` is changed to cluster scoped and the namespace field has no effect now.

Reply:
It just means where the config is stored. But I am ok either way.